.idea/String-Matching-Without-Scan-or-Regex.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you probably want to add `.idea/` to your `.gitignore` to prevent these editor configs from getting added